src/commands/sentinel-pending-scripts.json

```diff
@@ -10,6 +10,43 @@
             "ADMIN",
             "SENTINEL",
             "ONLY_SENTINEL"
-        ]
+        ],
+        "reply_schema": {
+            "type": "array",
+            "description": "List of pending scripts.",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "argv": {
+                        "type": "array",
+                        "description": "Script arguments.",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "flags": {
+                        "type": "string",
+                        "description": "Script flags."
+                    },
+                    "pid": {
+                        "type": "string",
+                        "description": "Script pid."
+                    },
+                    "run-time": {
+                        "type": "string",
+                        "description": "Script run-time."
+                    },
+                    "run-delay": {
```

Comment:
run-time and run-dalay should be one of them, they can not appear at same time; and they are integer value as well

Reply:
yes, i know... this is a map, how should i make a oneOf?
```
        if (sj->flags & SENTINEL_SCRIPT_RUNNING) {
            addReplyBulkCString(c,"run-time");
            addReplyBulkLongLong(c,mstime() - sj->start_time);
        } else {
            mstime_t delay = sj->start_time ? (sj->start_time-mstime()) : 0;
            if (delay < 0) delay = 0;
            addReplyBulkCString(c,"run-delay");
            addReplyBulkLongLong(c,delay);
        }
```

Reply:
ok, leave it like this is ok